base_layer/transactions/src/transaction.rs

```diff
@@ -806,4 +814,54 @@ mod test {
             "988ed705e6509684eb78ba81cd49525692002ab4dc79025bfd3fc051e45eb0b2"
         )
     }
+
+    #[test]
+    fn check_timelocks() {
+        let mut rng = rand::OsRng::new().unwrap();
+        let factories = CryptoFactories::new(32);
+        let k = BlindingFactor::random(&mut rng);
+        let v = PrivateKey::from(2u64.pow(32) + 1);
+        let c = factories.commitment.commit(&k, &v);
+        let s = PrivateKey::from_hex("6c6eebc5a9c02e1f3c16a69ba4331f9f63d0718401dea10adc4f9d3b879a2c09").unwrap();
+        let r = PublicKey::from_hex("28e8efe4e5576aac931d358d0f6ace43c55fa9d4186d1d259d1436caa876d43b").unwrap();
+        let sig = Signature::new(r, s);
+        let excess = Commitment::from_hex("9017be5092b85856ce71061cadeb20c2d1fabdf664c4b3f082bf44cf5065e650").unwrap();
+        let k = KernelBuilder::new()
```

Comment:
The `create_test_kernel` helper function could be used here.

Reply:
+1 for using the helper functions to reduce noise in tests

Reply:
Done